appeals/web/src/server/lib/mappers/appellantCase.mapper.js

```diff
@@ -387,7 +387,11 @@ export function initialiseAndMapData(appellantCaseData, appealDetails, currentRo
 					text: 'Site ownership'
 				},
 				value: {
-					text: appellantCaseData.siteOwnership.isFullyOwned ? 'Fully owned' : 'Partially owned'
+					text: appellantCaseData.siteOwnership.ownsAllLand
```

Comment:
the nested ternary is a little hard to read here - brackets or indentation could make it clearer